crates/compiler/builtins/roc/Bool.roc

```diff
@@ -32,6 +32,8 @@ Eq has
     ## cannot derive `isEq` for types that contain functions.
     isEq : a, a -> Bool | a has Eq
 
+## Represents the boolean truthy and falsy using an opaque type.
```

Comment:
? :smile: 
truthy is more fun than `True` but I also think it might confuse translation software

Reply:
I can update this in the next round of updates, if we want to save CI.

Reply:
Updated as requested